### PR TITLE
[plugin.audio.kxmxpxtx.bandcamp] v0.0.2

### PR DESCRIPTION
**Pull request by Virusmater**
### Description
https://bandcamp.com
Discover new artists and browse your collection
### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.audio.kxmxpxtx.bandcamp/.gitignore [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Byte-compiled / optimized / DLL files
```
Comment (basrieter) [inline plugin.audio.kxmxpxtx.bandcamp/.gitignore:1]: Remove from submission pls

File: plugin.audio.kxmxpxtx.bandcamp/resources/lib/cache/storageserverdummy.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+"""
```
Comment (basrieter) [inline plugin.audio.kxmxpxtx.bandcamp/resources/lib/cache/storageserverdummy.py:1]: Exclude from submission. Debug files?

Comment (Virusmater) [reply plugin.audio.kxmxpxtx.bandcamp/resources/lib/cache/storageserverdummy.py:1]: No, this is required to include as per:
https://kodi.wiki/view/Add-on:Common_plugin_cache

> Remember to include a fallback storageserverdummy.py file(Copy it from the script.common.plugin.cache/lib/ directory)

File: plugin.audio.kxmxpxtx.bandcamp/.gitignore (hunk 1 above)
Comment (Virusmater) [reply plugin.audio.kxmxpxtx.bandcamp/.gitignore]: done

**Comment by Virusmater** [comment]
Removed .gitignore
storageserverdummy is required by plugin design

**Pull request merged**